src/zarr/abc/codec.py

```diff
@@ -37,27 +34,6 @@
 CodecInput = TypeVar("CodecInput", bound=NDBuffer | Buffer)
 CodecOutput = TypeVar("CodecOutput", bound=NDBuffer | Buffer)
 
-TName = TypeVar("TName", bound=str, covariant=True)
-
-
-class CodecJSON_V2(TypedDict, Generic[TName]):
-    """The JSON representation of a codec for Zarr V2"""
```

Comment:
Probably should reexport these types here for backwards compatibility.

Reply:
done in https://github.com/zarr-developers/zarr-python/pull/3400/commits/30d48a8b7768b8fab33edef62a79eebcefaae8cf